.github/workflows/ci.yml

```diff
@@ -122,35 +122,3 @@ jobs:
           # Install depedencies at our cache location
           mv ../target/thumbv7em-none-eabihf/doc src/firmware
           mdbook build
-
-  hitl-trigger:
-    runs-on: ubuntu-latest
-    environment: hitl
-    if: ${{ github.event_name == 'merge_group' }}
```

Comment:
I think this condition here is critical to correct operation of the HITL + merge queue setup. If we don't have this condition, the job isn't marked as "Skipped" on a PR, but rather just never runs. With that setup, we can never actually ad it to the merge queue